examples/restaurant_demo/README.md

```diff
@@ -0,0 +1,279 @@
+
+Visit data for a week is stored in the `week_data.csv` file.
+
+The results are illustrated in the image below.
+
+![Counts per week day](img/counts_per_day.png)
+
+As you can see, the private values slightly differ from the actual ones but the
+overall trend is preserved.
+
+Now, let’s take a closer look at the technical details. Speaking in terms of
+*partitions* and *contributions*, the resulting bar chart has 7 partitions: one
+for each day of the week. A visitor may enter the restaurant once a day and
+hence contribute to a partition at most once. A visitor may enter the restaurant
+several times a week and hence contribute to up to 7 partitions. The code below
+uses `Count` to calculate the differentially private count of visits for a
+single day.
+
+```
+// Number of days a visitor may contribute to is limited to 3. All exceeding
+// visits will be discarded.
+private static final int COUNT_MAX_CONTRIBUTED_DAYS = 3;
+LN_3 = math.log(3)
+day_visits = bound_visits_per_week(self._day_visits, COUNT_MAX_CONTRIBUTED_DAYS)
+// Construct DP Count.
+if not epsilon:
+            x = Count(
+                epsilon=self._epsilon,
+                l0_sensitivity=COUNT_MAX_CONTRIBUTED_DAYS,
+                dtype="int",
+            )
+        else:
+            x = Count(
+                epsilon=epsilon, l0_sensitivity=COUNT_MAX_CONTRIBUTED_DAYS, dtype="int"
+            )
+```
+
+### Bounding the number of contributed partitions
+
+The parameter `COUNT_MAX_CONTRIBUTED_DAYS` defines the maximum number of
+partitions a visitor may contribute to. You might notice that the value of
+`COUNT_MAX_CONTRIBUTED_DAYS` in our example is 3 instead of 7. Why is that?
+Differential Privacy adds some amount of random noise to hide contributions of
+an individual. The more contributions an individual has, the larger the noise
+is. This affects the utility of the data. In order to preserve the data utility,
+you have to made an approximate estimate of how many times a week a person may visit a
+restaurant on average, and assumed that the value is around 3 instead of scaling
+the noise by the factor of 7.
+
+You have also pre-processed the input data and discarded all exceeding visits. The
+preprocessing is done by `bound_visits_per_week(self._day_visits, COUNT_MAX_CONTRIBUTED_DAYS)`. It is important
+to keep in mind that the library allows you to specify maximum amount of
+contributions, but doesn't validate that it is respected.
+
+Next, analyse for the remaining three cases:
+
+* Sum-up revenue per day of the week
+* Sum-up revenue per day of the week with preaggregation
+
+## Sum-up revenue per day of the week
+
+The previous example demonstrates how the contributed partitions are bounded.
+Now, you will demonstrate how individual contributions are clamped. Imagine Alice
+decides to calculate the sum of the restaurant revenue per week day in a
+differentially private way. For this she needs to sum up the visitors’ daily
```

Comment:
line 187: pls change it to "differentially private way. For this, she needs to sum up the **visitor's** daily"